app/jenkins-for-jira-ui/src/common/util/jenkinsConnectionsUtils.ts

```diff
@@ -12,6 +12,27 @@ export const setName = async (event: React.ChangeEvent<HTMLInputElement>, setSer
 	setServerName(event.target.value);
 };
 
+// TODO kill off isFormValid when rollout is complete
+export const isValid = (
```

Comment:
If its just for servername can we rename it isValidServerName or some such, if its a generic one we might want to reconsider the 
if (value.length > 100) being applied to everything

Reply:
Nah it's just for server name. Will update.